### DISCUSSION TITLE
Do you need to let the script run continuously?

### DISCUSSION THREAD

**Discussion by jon9595** [jdholtz/auto-southwest-check-in#51] [Q&A] [answered]
Hey! I was able to enter my confirmation number with my first and last name and the script found my flight. Do I need to let it stay running until the check-in time (tomorrow at 6:05 ET)? Or is it somehow scheduled in the background? Also if I used my confirmation number + first/last name, will it check in the other person I have on my reservation?

**Comment by jdholtz** [comment, marked as answer]
> Do I need to let it stay running until the check-in time

Yes. If you run the script directly (in the foreground), you will need to keep the terminal active. However, you can run the script in the background using your shell, Docker, etc. If you restart your computer, you will need to re-execute the script (Docker can do this automatically for you).

> Also if I used my confirmation number + first/last name, will it check in the other person I have on my reservation?

Yes. This is also mentioned on the README.
> Note: The script will check the entire party in under the same reservation, so there is no need to create more than one instance of the script per reservation.

**Reply by jon9595** [reply]
Thank you for the reply! And yep I'm seeing that in the README. Just was a bit confused on how it was phrased, but I appreciate this library. I did not want to wake up at 6:05 in the morning to check in. Definitely want to contribute back to this project if you have any outstanding features you are wanting to add!

**Reply by jdholtz** [reply]
> Definitely want to contribute back to this project if you have any outstanding features you are wanting to add!

That’d be great! I don’t have any big features I want to add currently. However, you can pay attention to the bug tracker for any issues to fix and/or feature requests from other users.
